backend/Pipfile

```diff
@@ -56,6 +56,7 @@ numpy = "*"
 inflection = "*"
 cybersource-rest-client-python = "*"
 pyjwt = "*"
+freezegun = "*"
```

Comment:
Used in the unit tests to simulate 12 hours passing after an event's creation. [Here's the library's repo](https://github.com/spulec/freezegun).

Reply:
FYI, you might have used `pipenv install` which updates packages where possible. For example, ruff is now pinned to 0.4.1 in the lock file (which may be breaking with previous versions). This is probably not your intended behavior, so you might want to revert and do a `pipenv upgrade` instead 

[Reading](https://stackoverflow.com/questions/55592586/how-to-install-update-package-with-pipenv-without-updating-the-rest-of-package)

Reply:
Thanks for the link, was helpful. Fixed.